docs/source/starter/lightning_lite.rst

```diff
@@ -0,0 +1,719 @@
+###########################################
+LightningLite - Stepping Stone to Lightning
+###########################################
+
+
+.. image:: https://pl-public-data.s3.amazonaws.com/docs/static/images/lite/lightning_lite.gif
+    :alt: Animation showing how to convert a standard training loop to a Lightning loop
+    :width: 600px
+    :align: center
+
+|
+
+:class:`~pytorch_lightning.lite.LightningLite` enables pure PyTorch users to scale their existing code
+on any kind of device while retaining full control over their own loops and optimization logic.
+
+:class:`~pytorch_lightning.lite.LightningLite` is the right tool for you if you match one of the two following descriptions:
+
+- I want to quickly scale my existing code to multiple devices with minimal code changes.
+- I would like to convert my existing code to the Lightning API, but a full path to Lightning transition might be too complex. I am looking for a stepping stone to ensure reproducibility during the transition.
+
+
+.. note:: LightningLite is a beta feature and will be considered stable for v1.6.
```

Comment:
```suggestion
.. warning:: `LightningLite` is currently a beta feature. API is subject to change.
```

Make this a warning. I wouldn't mention when this will be stable in case we need to change. We can still target this internally.

Reply:
Done !